e2e/cypress/support/commands.js

```diff
@@ -134,3 +135,84 @@ Cypress.Commands.add('deleteMetadata', (metadataTrackName) => {
 
   log.end();
 });
+
+Cypress.Commands.add('randomizeSampleName', (samplePosition) => {
```

Comment:
I would rename this to `getRandomSampleName`

Reply:
Hmm... it doesn't really return or "get" anything, so I don't think that's a fitting name?

Reply:
ah ok, I thought it did